lib/css_parser/rule_set.rb

```diff
@@ -142,7 +142,7 @@ def replace_declaration!(property, replacements, preserve_importance: false)
 
         if preserve_importance
           importance = get_value(property).important
-          replacement_declarations.each { |_key, value| value.important = importance }
+          replacement_declarations.each_value { |value| value.important = importance }
```

Comment:
probably this is the only change in production code that can have effect, but specs pass and I think that `def_delegators` is doing the job here